docs/UserGuide.md

```diff
@@ -32,103 +33,138 @@
 > **Command Format**
 > * Words in `UPPER_CASE` are the parameters.
 > * Items in `SQUARE_BRACKETS` are optional.
-> * Items with `...` after them can have multiple instances.
 > * The order of parameters is fixed.
 
 #### Viewing help : `help`
 Format: `help`
 
 > Help is also shown if you enter an incorrect command e.g. `abcd`
  
-#### Adding a person: `add`
-Adds a person to the address book<br>
-Format: `add NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]...` 
+#### Adding an item: `add`
+Adds an item into the task manager<br>
+
+*With deadline*
+
+Format: `add NAME by DUE_DATE DUE_TIME [repeat every RECURRING_INTERVAL]`
+>Notice the `by` keyword? We use `by` to denote a deadline.
```

Comment:
One way to do it:
Look for the last "by" in the sentence.
Check if the string following "by" is a supported date/time format.
If not, it will be a floating task.
Cons of this: If user tried to type in a date time format but typos, it will be treated as a floating task. But we have undo to remedy this, so should be ok